src/test/java/core/basesyntax/model/FruitTransactionTest.java

```diff
@@ -0,0 +1,50 @@
+package core.basesyntax.model;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertTrue;
+
+import core.basesyntax.model.FruitTransaction.Operation;
+import org.junit.Test;
+
+public class FruitTransactionTest {
+    private static String FRUIT = "apple";
+    private static String OPERATION_CODE = "b";
+    private static String ILLIGAL_OPERATION_CODE = "w";
+    private static Operation TEST_BALANCE_OPERATION = FruitTransaction.Operation.BALANCE;
+    private FruitTransaction fruitTransaction;
+
+    @Test
+    public void getExistingOperation_ok() {
```

Comment:
you should follow pattern 
`<method name>_<description>_<expected behaviour>()`